models/topics-models.js

```diff
@@ -0,0 +1,9 @@
+const db = require("../db/connection");
+
+exports.selectTopics = () => {
+  console.log("bye");
```

Comment:
Remember to remove any console.logs before submitting a pull request - you don't want console.logs (especially those used for debugging!) in production code.